config/secure-headers.php

```diff
@@ -3,319 +3,515 @@
+	 * @see https://w3c.github.io/webappsec-referrer-policy
 	 *
 	 * Available Value: 'no-referrer', 'no-referrer-when-downgrade', 'origin', 'origin-when-cross-origin',
 	 *                  'same-origin', 'strict-origin', 'strict-origin-when-cross-origin', 'unsafe-url'
 	 */
-
 	'referrer-policy' => 'no-referrer',
 
-	/*
-	 * Clear-Site-Data
+	/**
+	 * Cross-Origin-Embedder-Policy.
+	 *
+	 * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy
+	 *
+	 * Available Value: 'unsafe-none', 'require-corp', 'credentialless'
+	 */
+	'cross-origin-embedder-policy' => 'unsafe-none',
+
+	/**
+	 * Cross-Origin-Opener-Policy.
+	 *
+	 * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Opener-Policy
+	 *
+	 * Available Value: 'unsafe-none', 'same-origin-allow-popups', 'same-origin'
+	 */
+	'cross-origin-opener-policy' => 'unsafe-none',
+
+	/**
+	 * Cross-Origin-Resource-Policy.
+	 *
+	 * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Resource-Policy
 	 *
-	 * Reference: https://w3c.github.io/webappsec-clear-site-data/
+	 * Available Value: 'same-site', 'same-origin', 'cross-origin'
 	 */
+	'cross-origin-resource-policy' => 'cross-origin',
 
+	/**
+	 * Clear-Site-Data.
+	 *
+	 * @see https://w3c.github.io/webappsec-clear-site-data/
+	 */
 	'clear-site-data' => [
 		'enable' => false,
 		'all' => false,
 		'cache' => true,
+		'clientHints' => true,
 		'cookies' => true,
 		'storage' => true,
 		'executionContexts' => true,
 	],
 
-	/*
-	 * HTTP Strict Transport Security
+	/**
+	 * HTTP Strict Transport Security.
 	 *
-	 * Reference: https://developer.mozilla.org/en-US/docs/Web/Security/HTTP_strict_transport_security
+	 * @see https://developer.mozilla.org/en-US/docs/Web/Security/HTTP_strict_transport_security
 	 *
 	 * Please ensure your website had set up ssl/tls before enable hsts.
 	 */
-
 	'hsts' => [
 		'enable' => env('SECURITY_HEADER_HSTS_ENABLE', false),
 		'max-age' => 15552000,
 		'include-sub-domains' => false,
+		'preload' => false,
+	],
+
+	/**
+	 * Reporting Endpoints.
+	 *
+	 * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Reporting-Endpoints
+	 *
+	 * Note: The array key is the endpoint name, and the value is the URL.
+	 */
+	'reporting' => [
+		// 'csp' => 'https://example.com/csp-reports',
+		// 'nel' => 'https://example.com/nel-reports',
 	],
 
-	/*
-	 * Expect-CT
+	/**
+	 * Network Error Logging.
 	 *
-	 * Reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expect-CT
+	 * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Network_Error_Logging
+	 * @see https://developer.mozilla.org/en-US/docs/Web/API/Reporting_API
 	 */
+	'nel' => [
+		'enable' => false,
+		// The name of reporting API, not the endpoint URL.
+		'report-to' => '',
+		'max-age' => 86400,
+		'include-subdomains' => false,
+		'success-fraction' => 0.0,
+		'failure-fraction' => 1.0,
+	],
 
+	/**
+	 * Expect-CT.
```

Comment:
Probably better to keep with the copy, but this is deprecated.

Reply:
yeah :/